csaf_2.1/prose/edit/src/tests-04-presets.md

```diff
@@ -0,0 +1,72 @@
+## Presets
+
+A test preset is predefined set of tests that was given a name.
+It MAY contain any number of tests.
+Two presets MAY overlap.
+The content of a preset MAY be different.
```

Comment:
What do you mean?

Reply:
Resolved in [26cd298](https://github.com/oasis-tcs/csaf/pull/1005/commits/26cd2984a0d771b588a1d64abeda62f8c7403f8e)